src/rtFileDownloader.cpp

```diff
@@ -906,20 +906,26 @@ bool rtFileDownloader::downloadFromNetwork(rtFileDownloadRequest* downloadReques
     /* check for errors */
     if (res != CURLE_OK)
     {
-        stringstream errorStringStream;
+        rtString errorString("Download error for: ");
+        errorString.append(downloadRequest->fileUrl().cString());
 
-        errorStringStream << "Download error for: " << downloadRequest->fileUrl().cString()
-                << ".  Error code : " << res << ".  Using proxy: ";
+        errorString.append(". Error code : ");
+        // keep stringstream only for errorcode
+        stringstream errorCode("");
```

Comment:
If we're looking to avoid stringstreams, then let's remove this one too.

Reply:
made the changes